changelog.md

```diff
@@ -32,36 +36,47 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
     `DicomCollection.from_directory(directory_path)` which would pull all DICOM
     files nested within a directory and sort them into `DicomStudy` objects
     based on their header UIDs.
+- Added CLI commands for a WIP docker server, logfile orchetration, and DICOM
+  editor tools.
+- Added a range of xlwings tools that allow the use of PyMedPhys functions
+  within Excel
+- Added the rudementry code enabling the pulling of profiles from Mephysto
```

Comment:
```suggestion
- Added rudimentary code to pull profiles from Mephysto
```

Reply:
Thanks Matt, I think I should find a VSCode extension for spell check...